tox.ini

```diff
@@ -120,6 +120,43 @@ max-complexity = 21
 
 
 ##
+# Mypy linting
+##
+
+[testenv:mypy]
+
+basepython = python3.5
```

Comment:
Why not 3.6?

Reply:
3.6 isn't currently in Klein's test matrix.  It should be, but that's another PR, I think.

Reply:
#218 for tracking.